fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/syntax/Syntax2.scala

```diff
@@ -92,6 +98,27 @@ object Syntax2 {
     @inline final def fromEither[E1 >: E, A1](implicit ev: A <:< Either[E1, A1]): F[E1, A1] = F.flatMap[Any, E1, A, A1](r)(F.fromEither[E1, A1](_))
     @inline final def fromOption[E1 >: E, A1](errorOnNone: => E1)(implicit ev1: A <:< Option[A1]): F[E1, A1] = F.flatMap[Any, E1, A, A1](r)(F.fromOption(errorOnNone)(_))
 
+    @inline final def retryUntil(f: E => Boolean): F[E, A] =
+      F.retryUntil(r)(f)
+
+    @inline final def retryUntilM(f: E => F[Nothing, Boolean]): F[E, A] =
+      F.retryUntilM[Any, E, A](r)(f)
+
+    @inline final def retryWhile(f: E => Boolean): F[E, A] =
+      F.retryWhile(r)(f)
+
+    @inline final def retryWhileM(f: E => F[Nothing, Boolean]): F[E, A] =
+      F.retryWhileM[Any, E, A](r)(f)
+
+    @inline final def someOrElse[B](default: => B)(implicit ev: A <:< Option[B]): F[E, B] =
```

Comment:
This is the same as `fromOption` here - https://github.com/7mind/izumi/pull/1390/files#diff-9018879521a7e6653abbe00676af3367f1b5c0ed3293d54f424fa03e04aa3d8eR99